src/CMakeLists.txt

```diff
@@ -8,9 +8,9 @@ add_library(quicr
 
 # Suppress external lib warnings
 set_property(GLOBAL PROPERTY RULE_MESSAGES OFF)
-target_compile_options(picotls-core PRIVATE -w)     # TODO Should move to qtransport
-target_compile_options(picotls-openssl PRIVATE -w)  # TODO Should move to qtransport
-target_compile_options(picoquic-core PRIVATE -w)    # TODO Should move to qtransport
+target_compile_options(picotls-core PRIVATE -w)
+target_compile_options(picotls-openssl PRIVATE -w)
+target_compile_options(picoquic-core PRIVATE -w)
```

Comment:
Should these no longer move to qtransport?

Reply:
Turns out it has to be in each repo :) Not just transport.